build/myget-release.yml

```
@@ -0,0 +1,132 @@
+name: $(date:yyyyMMdd)$(rev:.r)
+
+trigger: none
+
+variables:
+  - group: 'Arcus Event Grid - Integration Testing'
+  - group: 'Arcus - GitHub Package Registry'
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.105'
+  - name: 'Project'
+    value: 'Arcus.EventGrid'
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
+            displayName: 'Compile'
+            inputs:
+              projects: 'src/*.sln'
+              arguments: '--configuration $(Build.Configuration)'
```

You are not stamping it with the correct suffix
```suggestion
              arguments: '--configuration $(Build.Configuration)' --version-suffix $(packageVersion)
```